src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -850,13 +851,33 @@ In the Job xml schema it is available as `preciousResult`:
 
 ==== Job Result Map
 
-Job has key-value map associated to it, which is called *Job Result Map*. Each task can contribute to this *Job Result Map*. Consider the following groovy example:
+Job has key-value map associated to it, which is called *Job Result Map*.
```

Comment:
In addition to the Job Results List, a Job can also store some results into a dedicated key-value map, which is called *Job Result Map*.

Every tasks can contribute to ...